### PR TITLE
Updated internal links

### PR DESCRIPTION
**Pull request by Vladikasik**

### CODE REVIEW

**Comment by Apolotoybo** [comment]
Access

**Comment by MarketBeatAd** [comment]
d8c619fd696c2d09a49f1393522ce6de18c15f85

**Comment by Akinayeeen16** [comment]
help

**Review by airaojekhile — APPROVED** [review]

**Comment by MarketBeatAd** [comment]
Main

File: README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
 
 * [Definitions](#definitions)
```
Comment (MarketBeatAd) [inline README.md:5]: Update

**Review by MarketBeatAd — APPROVED** [review]

**Review by Makeuptehrani — APPROVED** [review]

**Review by Masw21 — COMMENTED** [review]
Update

**Review by Makeuptehrani — APPROVED** [review]

**Pull request merged**